README.md

```diff
@@ -47,7 +57,7 @@ While [different opinions](https://scifi.stackexchange.com/questions/192022/what
 
 ## Requirements
 
-- Xcode 10.2+ and Swift 5.0+
+- Xcode 10.x and Swift 5.0-5.2
```

Comment:
Actually, Xcode 11.x is also possible as Accio is only deprecated starting with Xcode 12

Reply:
You are totally right, of course. Fixed via d8d186fd466fe89424fa8dd6fa0060372aec1518.